src/config.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 Espresso Systems (espressosys.com)
+// This file is part of the Configurable Asset Privacy (CAP) library.
+
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version. This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+// details. You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Configuration
+
+use ark_ec::{PairingEngine, TEModelParameters};
+use ark_ff::{PrimeField, SquareRootField};
+
+/// Configuration for CAP system
+pub trait CapConfig {
```

Comment:
do we also need to configure which hash function, encryption and signature scheme here?

Reply:
I don't think so. Our scheme are not generic over choice of hash function, encryption etc. we have specific, fixed choice.